src/app/screens/Home/styles.module.scss

```diff
@@ -1,16 +1,103 @@
 @import 'scss/variables/_sizes.scss';
 @import '_colors.scss';
 
-.container {
+$calculator-color: #b2e2d4;
+$display-box-color: #6abeb3;
+
+.calculatorContainer {
   align-items: center;
+  background-color: $calculator-color;
+  border-radius: 10%;
   display: flex;
   flex-direction: column;
   height: calc(100% - #{$topbar-heigth});
   justify-content: flex-start;
+  margin: 20px auto;
+  width: 600px;
+}
+
+.calculatorButton {
+  background-color: $calculator-color !important;
+  border: 3px solid white;
+  border-radius: 100px;
+  color: rgb(97, 91, 91) !important;
```

Comment:
declare this color in the same place as $calculator-color

Reply:
ok